Content.Server/GameObjects/Components/Chemistry/SolutionComponent.cs

```diff
@@ -281,7 +281,26 @@ void IExamine.Examine(FormattedMessage message)
             {
                 if (_prototypeManager.TryIndex(reagent.ReagentId, out ReagentPrototype proto))
                 {
-                    message.AddText($"{proto.Name}: {reagent.Quantity}u\n");
+                    if (inDetailsRange)
+                    {
+                        message.AddText($"{proto.Name}: {reagent.Quantity}u\n");
+                    }
+                    else
+                    {
+                        //This is trash but it shows the general idea 
+                        var colour = proto.SubstanceColor;
```

Comment:
I know "colour" is correct, but AE is better for code.

Reply:
> citation needed

I'll change it though, since it seems even the other non-americans are bowing down to your tyrannical rule.